dev-support/create-release/do-release-docker.sh

```diff
@@ -173,6 +176,12 @@ if [ -n "$JAVA" ]; then
   JAVA_VOL=(--volume "$JAVA:/opt/hbase-java")
 fi
 
+MAVEN_MOUNT=()
+if [ -n "${MAVEN_VOLUME}" ]; then
+  MAVEN_MOUNT=(--mount "type=volume,src=${MAVEN_VOLUME},dst=/home/hbase-rm/.m2-repository/")
+  echo "REPO=/home/hbase-rm/.m2-repository" >> "${ENVFILE}"
```

Comment:
s/hbase-rm/${USER}/ ?

Reply:
Or, the whole point is a clean dir into which the build can populate a new .m2 repository or is it that you want to point the build at an existing populated local repo?

Reply:
oh. this is in the container? And hbase-rm is a container user? You are trying to share the repo across build stages?  ok.